refs/biblio.json

```diff
@@ -1088,6 +1088,18 @@
     "FLEXBOX": {
         "aliasOf": "css3-flexbox"
     },
+    "FLEXFEC": {
+      "title": "RTP Payload Format for Flexible Forward Error Correction (FEC)",
+      "href": "https://tools.ietf.org/html/draft-ietf-payload-flexible-fec-scheme",
+      "authors": [
+        "V. Singh",
+        "A. Begen",
+        "M. Zanaty",
+        "G. Mandyam"
+      ],
+      "status": "3 July 2017. Internet Draft (work in progress)",
```

Comment:
Can you please split the status into `status` and `rawDate` (YYYY-MM-DD) fields?

Reply:
Done.